src/cpp/server_core/ServerDatabase.cpp

```diff
@@ -48,7 +48,7 @@ constexpr const char* kDefaultSqliteDatabaseDirectory = "/var/lib/rstudio-server
 constexpr const char* kDatabaseHost = "host";
 constexpr const char* kDefaultDatabaseHost = "localhost";
 constexpr const char* kDatabaseName = "database";
-constexpr const char* kDefaultDatabaseName = "rstudio";
+constexpr const char* kDefaultDatabaseName = "rstudio-os";
```

Comment:
This is going to be easy to mess up by accident when resolving an unrelated merge conflict in the future. How would you feel about having both schema names as constants, and deciding which one to use at runtime based on `isProfessionalEdition()`? 

(Alternately, we could make this an option and have it use a different default based on `isProfessionalEdition()`)